internal/controller/model_controller.go

```diff
@@ -297,10 +297,8 @@ func (r *ModelReconciler) modellerJob(ctx context.Context, model, baseModel *api
 					ServiceAccountName: modellerServiceAccountName,
 					Containers: []corev1.Container{
 						{
-							Name:  containerName,
-							Image: model.GetImage(),
-							// NOTE: tini should be installed as the ENTRYPOINT the image and will be used
```

Comment:
Note: we no longer inject tini during an in-cluster git build, so I removed the comments